components/Front/index.js

```diff
@@ -132,6 +134,27 @@ const Front = ({
   finite,
   hasOverviewNav
 }) => {
+  const now = new Date()
+  const dailyUpdateTime = new Date(
+    now.getFullYear(),
+    now.getMonth(),
+    now.getDate(),
+    5
+  )
+  const shouldRefetch =
+    front.meta.path === '/' &&
```

Comment:
This might be undefined during loading and will make boom. While `front?.meta?.path` would be an option this would further complicate things because we'd have to wait for the page to load and path to be defined before we know it. I'd switch to using the same expression as in the query var which is always already defined on load:
```
const path = props.path || cleanAsPath(props.router.asPath)
```